Cargo.lock

```diff
@@ -826,7 +826,7 @@ dependencies = [
 [[package]]
 name = "matrix-sdk-common"
 version = "0.7.0"
-source = "git+https://github.com/matrix-org/matrix-rust-sdk#75683d268fc2e0fc1c21e038ac9c59706a9b9180"
+source = "git+https://github.com/matrix-org/matrix-rust-sdk#70bcddfba5e19f2d0fd11577edb55b9868f89678"
```

Comment:
needs changelog update

Reply:
Thanks, sorry I messed around with this because the workaround fix came after I'd already made one release.

Fixed in d6cf7f5